django/db/models/expressions.py

```diff
@@ -456,7 +456,7 @@ class TemporalSubtraction(CombinedExpression):
     def __init__(self, lhs, rhs):
         super(TemporalSubtraction, self).__init__(lhs, self.SUB, rhs, output_field=fields.DurationField())
 
-    def as_sql(self, compiler, connection):
+    def as_sql(self, compiler, connection, **kwargs):
```

Comment:
I'm still uncertain about the purpose or value in adding unused `**kwargs` in places like this?